src/CarbonAware.NewCLI/README.md

```diff
@@ -0,0 +1,51 @@
+The Carbon Aware SDK provides a CLI to get the marginal carbon intensity for a given location and time period. The values reported in the Green Software Foundation's specification for marginal carbon intensity (Grams per Kilowatt Hour). In order to use the API, the environment needs to be prepared with a set of configuration parameters. Instructions on setting up the environment could be found here - https://github.com/microsoft/carbon-aware-sdk/blob/dev/GettingStarted.md
+
+# Carbon Aware CLI Commands
+- [Carbon Aware CLI Commands](#carbon-aware-cli-commands)
+  - [Command Keywords](#command-keywords)
+    - [`emissions`](#emissions)
+      - [`list`](#list)
+  
+## Command Keywords
+
+### `emissions`
+
+The `emissions` keyword is used to invoke any commands related to getting Carbon Emissions data.
+
+#### `list`
+
+**Inputs:**
+
+| Keyword       | Description                         | Optional? | Default | Example                                           |
+| ------------- | :---------------------------------- | :-------- | :------ | :------------------------------------------------ |
+| `--locations` | Space separated list of locations.  | No        | N/A     | `--locations "useast" "uswest"`                   |
+| `--startTime` | Start time for emissions data       | Yes       | `null`  | `--startTime "2022-05-17T20:45:11.5092741+00:00"` |
+| `--toTime`    | Ending time for emissions data.     | Yes       | `null`  | `--toTime "2022-05-17T20:45:11.5092741+00:00"`    |
+| `--best`      | Bool Filter to give the best result | Yes       | `false` | `--best true`                                     |
```

Comment:
Rather than calling this a `Bool Filter` it may be easier to use the term `flag` or `boolean flag` and have the example simply show `--best`

Reply:
After putting some more thought into this, I dont think `--best` should be part of the `list' command since it always returns one element. So, perhaps it should be its own command - `emissions best show`. @bderusha @KanishkT123 @JenMadiedo  thoughts?

Reply:
+1. We already know that the aggregator command is different, making it a different subcommand entirely makes sense to me.